app/views/reports-list/report-form-fields.njk

```diff
@@ -0,0 +1,44 @@
+{# report-form-fields.njk #}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">{{ reportTitle }}</h1>
+    <div class="govuk-form-group">
+      <label class="govuk-label govuk-label--s" for="schemeId">
+        Select the scheme to view data for
+      </label>
+      <select class="govuk-select" id="schemeId" name="schemeId">
+        {% for scheme in schemes %}
+          <option value="{{scheme.schemeId}}">{{scheme.name}}</option>
+        {% endfor %}
+      </select>
+    </div>
+    <div class="govuk-form-group">
+      <label class="govuk-label govuk-label--s" for="year">
```

Comment:
I assume these classes are set this way in the matching stylesheet, however the --s in GDS is -s, if that makes sense to change or not is subjective.

Reply:
amended to -s and also corrected other places in the code where it was also --s